GOVERNANCE.md

```diff
@@ -0,0 +1,90 @@
+
+- Actively answer technical questions raised by community users in GitHub issues.
+- Help test the projects
+- Help review the pull requests (PRs) submitted by others
+- Help improve technical documents
+- Submit valuable issues
+- Report or fix known and unknown bugs
+- Write articles about source code analysis and usage cases for a project.
+- Give representations of Apollo topic in conferences.
+- Take part in our discussions of features, enhancements, etc.
+
+## Committers
+
+Committers are contributors who have shown that they are committed to the continued development of the project through ongoing engagement with the community and recognized by PMCs for their outstanding contributions.
+
+### How to become a Committer
+
+A Committer must have accomplished one or more of the following items:
+
+- Demonstrated a good sense of responsibility in PR reviews.
+- Demonstrated deep understanding of Apollo components by contributing significantly as:
+    - Finished 2 or more tasks of Medium difficulty
+    - Fixed 1 or more tasks of Hard difficulty
+- Nominated by one PMC member and gained more +1 than -1.
+
+### Privileges and responsibilities
+
+- Control overall code quality of projects
+- Guide Contributors to contribute to the community continuously
+- Participate in design discussions
+
+## Project Management Committee
+
+The PMC(Project Management Committee) functions as the core management team that oversees the Apollo community. The PMC has additional responsibilities over and above those of Committers. These responsibilities ensure the smooth running of the project.
+
+### How to become a PMC
+
+- Membership of the PMC is by invitation from the existing PMC members. 
+- A nomination will result in discussion and then a vote by the existing PMC members.
+- PMC membership votes are subject to consensus approval of the current PMC members.
+
+### Privileges and responsibilities
+
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Committers, and makes all major decisions for the future with respect to Apollo, such as project-level governance policies, management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached.
+
+# Decision making and voting
+
+Proposals and ideas can be submitted for agreement via a github issue or PR.
+
+Major changes such as feature proposals and organization or process changes should be brought to the PMC. For the change to happen, the change must earn the supermajority (2/3) votes.
+
+# Conflict resolution
+
+In general, we prefer that technical issues and other disputes upon which consensus can't be reached are amicably worked out between the persons involved. If a dispute cannot be decided independently, the PMC can be called in to resolve the issue by voting. The same PR can be used or a separate PR can be opened for voting.
```

Comment:
This is a great point, community governance is essentially by people participation, generally speaking, people management, good regulation is more important.